fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccrualBasedAccountingProcessorForLoan.java

```diff
@@ -418,6 +418,7 @@ private void createJournalEntriesForChargeback(LoanDTO loanDTO, LoanTransactionD
         final Long loanProductId = loanDTO.getLoanProductId();
         final Long loanId = loanDTO.getLoanId();
         final String currencyCode = loanDTO.getCurrencyCode();
+        final BigDecimal overpaidAmount = loanDTO.getOverpaidAmount();
```

Comment:
final BigDecimal overPaymentAmount = loanTransactionDTO.getOverPayment();

Reply:
chaged it to the suggested one